admin_guide/assembly_replace-master-host.adoc

```diff
@@ -17,7 +17,7 @@ You can replace a failed master host.
 
 First, remove the failed master host from your cluster, and then add a replacement
 master host. If the failed master host ran etcd, scale up etcd by adding etcd
-to the new master host.
+to the new master host. 
```

Comment:
@sdodson, if you replace a failed master host, is the etcd pod restored automatically, or do you need to scale etcd up on the master separately?

Reply:
etcd would needs to be scaled up as a separate step.